### ISSUE TITLE
Statistics of media that is shown in a liveshow

### ISSUE THREAD

**Issue by jamesagada** [KenT2/pipresents-gapless#150]
Is there an option for the statistics of which media are shown and at what time in a liveshow. The pp_log.txt when set with the -d option provides a lot of information but it is really a hassle to get the statistics out from there. Is there an option that shows only the media been shown and at what time?

**Comment by KenT2** [comment]
Yes, section 21.1 of the manual. To enable writing of statistics for every track uncomment line 355 of pp_gapshow.py
